pkg/apiserver/webhooks/validating-webhook/util.go

```diff
@@ -0,0 +1,22 @@
+package validatingwebhook
+
+import (
+	"sync"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+var kubeClient *kubernetes.Interface
```

Comment:
You sure you mean for this to be a pointer to an interface?  I think a regular interface will be fine.